src/main/java/entities/Timeslot.java

```diff
@@ -97,4 +97,9 @@ public boolean equals(Object o) {
         }
         return false;
     }
+
+    @Override
+    public int compareTo(Timeslot that) {
+        return this.start.compareTo(that.getStart());
```

Comment:
highly clutch addition. we should merge this soon so it can be used in other places